modyn/selector/internal/selector_strategies/abstract_selection_strategy.py

```diff
@@ -391,3 +391,12 @@ def _persist_samples(self, keys: list[int], timestamps: list[int], labels: list[
 
         for proc in processes:
             proc.join()
+
+    def seed_selector(self, seed: float) -> bool:
+        with MetadataDatabaseConnection(self._modyn_config) as database:
+            if database.drivername == "postgresql":
+                assert -1 <= seed <= 1
+                database.session.execute(func.setseed(seed))
```

Comment:
I see one potential problem here: You execute this function for this particular session that only exists in the seed_selector function. Is this setting _global_ for the entire database or local? Can you try whether creating a session, executing that function, creating a new session and getting the seed / at least some random number works correctly? I fear it might be per-session, in which case we need to move the seed to the modyn config and use it in the `MetadataDatabaseConnection` instead